source/content/guides/domains/05-custom-domains.md

```diff
@@ -94,6 +94,9 @@ Yes. Skipping HTTPS provisioning is no longer an option.
 ### Is Wild Card DNS routing supported by Domain Verification?
 Pantheon does not allow wild card domains to be directly added as a custom domain. Customers may point wildcard domains (eg: *.example.com) in their own DNS to Pantheon, but are still required to have specific domains (eg: mysite.example.com) added and connected to specific environments on Pantheon.
 
+### How can I know which domains are still pending ownership verification ?
+For any domain that has been added that is pending verification, clicking on the "Details" button in the Domains list page for that domain will take you to another page where you can put in the information required to verify ownership for that domain. If the ownership of the domain has been already verified, the detail page will instead show the DNS records you need to update in your authoritative DNS to point to Pantheon, as well as the status of HTTPS provisioning. In other words, if your domain is not verified, we will require you to provide the necessary information to verify ownership first. 
+
```

Comment:
@manoj-sarma should this FAQ point folks towards Terminus for how to check domain status for all custom domains connected to a given environment?  https://docs.pantheon.io/terminus/commands/https-info

Reply:
I think that's a great idea. Will add that.